library/src/main/java/org/apache/apex/malhar/lib/window/impl/KeyedWindowedOperatorImpl.java

```diff
@@ -52,81 +52,85 @@
 {
 
   @Override
-  protected Collection<Window.SessionWindow> assignSessionWindows(long timestamp, Tuple<KeyValPair<KeyT, InputValT>> inputTuple)
+  protected <T> Collection<Window.SessionWindow> assignSessionWindows(long timestamp, Tuple<T> inputTuple)
```

Comment:
The method signature here is the same as what it is in `AbstractWindowedOperator`, if I change to what you suggested it will no longer override the method in parent class. And in `AbstractWindowedOperator` there's no `keyT` type parameter so I cannot change the signature there either. Is there another way to do it?